README.mdown

```diff
@@ -39,6 +41,30 @@ curl -X POST -H "Content-Type: application/json" \
      http://127.0.0.1:8080/incoming/your-secret-token
 ```
 
+### UptimeRobot
+
+The URL is constructed as follows: `http(s)://<host>:<port>/incoming/uptimerobot/<your-secret-token>`
+
+#### Usage
+
+Send a GET request to the webhook URL with the follow query params.
+In UptimeRobot's settings, add the following webhook (the trailing ampersand
+is required):
```

Comment:
Can we link this? Also, I think this should mention something about hooks per monitor vs. account-wide hooks, or are those all on the same settings page?